data/example_partition_config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "partitions":[
+        {"id":0, "cat-ids":["cat-1","cat-2"], "nex-ids":["nex-1", "nex-2"], "remote-up":[], "remote-down":[ {"id":"nex-2", "partition":1 }]},
```

Comment:
I feel like this may or may not need to be changed so that the test reflect the output the partition generate will produce.  The structure is the same, but the values will be empty now.  Really, this is something that needs to be verified in expanded tests, as mentioned separately.